liberty-maven-plugin/src/main/resources/net/wasdev/wlp/maven/plugins/MvnMessages.properties

```diff
@@ -301,3 +301,7 @@ error.project.not.compile.useraction=Make sure the project has been compiled.
 error.app.dependency.not.found=CWWKM2175E: Failed to install loose application. The application dependency library {0} cannot be found.
 error.app.dependency.not.found.explanation=The application dependency cannot be found in Maven local repository.
 error.app.dependency.not.found.useraction=Rebuild the project by running mvn install again.
+
+error.install.dir.validate=CWWKM2100E: When installDirectory attribute is set, it must point to a directory that contains lib/ws-launch.jar.
```

Comment:
I am wondering if we need to even mention lib/ws-launch.jar.  We are using that as a sanity check jar but is really important for the message.  Would the following make more sense:

+error.install.dir.validate=CWWKM2100E: When installDirectory attribute is set, it must point to a directory that contains a Liberty installation.